packages/terra-application-header-layout/src/terra-dev-site/doc/ApplicationHeader.doc.jsx

```diff
@@ -1,15 +1,15 @@
 /* eslint-disable import/no-extraneous-dependencies, import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions */
 import React from 'react';
 import DocTemplate from 'terra-doc-template';
-import { name } from '../package.json';
-import ReadMe from '../docs/README.md';
+import { name } from '../../../package.json';
+import ReadMe from '../../../docs/README.md';
 
 // Component Source
-import HeaderLayoutSrc from '!raw-loader!../src/ApplicationHeaderLayout.jsx';
+import HeaderLayoutSrc from '!raw-loader!../../../src/ApplicationHeaderLayout.jsx';
 
 // Example File
-import HeaderWireframe from './index-examples/HeaderWireframe';
-import HeaderWireframeSrc from '!raw-loader!./index-examples/HeaderWireframe.jsx';
+import HeaderWireframe from './example/HeaderWireframe';
+import HeaderWireframeSrc from '!raw-loader!./example/HeaderWireframe.jsx';
```

Comment:
This needs to come from src

Reply:
This should have failed to compile, but there is a bug in my hot reloading logic.